### DISCUSSION TITLE
Unknown FCS component: note

### DISCUSSION THREAD

**Discussion by matheusdarocha** [JSBSim-Team/jsbsim#603] [General]
Good Morning,

I am running JSBSim in python and the message **"Unknown FCS component: note"** is showing at immediate window, which makes the debug really complicated.

Could somebody give me an idea how to track this unknown component please?

Thanks

**Comment by seanmcleod** [comment]
Looks like you've got an element named `note` in your xml.

https://github.com/JSBSim-Team/jsbsim/blob/375f5be0fdfeb210f4cf66f938e3e6fd426c2966/src/models/FGFCS.cpp#L580-L586

**Reply by matheusdarocha** [reply]
Thanks! It worked
